components/elm/bld/configure

```diff
@@ -493,20 +493,24 @@ sub write_filepath_cesmbld
 		     "external_models/emi/src/emi_data_definition",
 		     "external_models/emi/src/emi_data_exchange",
 		     "external_models/emi/src/emi_data_types",
+         "external_models/sbetr/src/betr/betr_bgc",
          "external_models/sbetr/src/betr/betr_core",
          "external_models/sbetr/src/betr/betr_dtype",
-         "external_models/sbetr/src/betr/betr_bgc",
+         "external_models/sbetr/src/betr/betr_echem",
          "external_models/sbetr/src/betr/betr_grid",
-         "external_models/sbetr/src/betr/betr_para",
          "external_models/sbetr/src/betr/betr_main",
          "external_models/sbetr/src/betr/betr_math",
+         "external_models/sbetr/src/betr/betr_para",
          "external_models/sbetr/src/betr/betr_rxns",
+         "external_models/sbetr/src/betr/betr_transport",
          "external_models/sbetr/src/betr/betr_util",
          "external_models/sbetr/src/driver/shared",
-         "external_models/sbetr/src/driver/alm",
+         "external_models/sbetr/src/driver/elm",
          "external_models/sbetr/src/Applications/app_util",
          "external_models/sbetr/src/Applications/soil-farm/bgcfarm_util",
-         "external_models/sbetr/src/Applications/soil-farm/CENT_ECACNP",
+         "external_models/sbetr/src/Applications/soil-farm/v1eca/v1ecaPara",
```

Comment:
Is it possible to avoid using the specific version of eca in the names?

Reply:
@minxu74 , It means to include a variety of model formulations. Otherwise, it may more confusion by removing this. Also, E3SM by default will not turn it on.